app/helpers/validation_helper.rb

```diff
@@ -73,22 +75,14 @@ module ValidationHelper # rubocop:todo Metrics/ModuleLength
 
   VALID_STATES = STATE_ABBREVIATIONS.values
 
+  ISOLATION_AVAILABLE = isolation_available?(default_playbook).freeze
+
   USER_SELECTABLE_MONITORING_REASONS = [
-    'Completed Monitoring',
-    'Meets criteria to shorten quarantine',
-    'Does not meet criteria for monitoring',
-    'Meets Case Definition',
-    'Lost to follow-up during monitoring period',
-    'Lost to follow-up (contact never established)',
-    'Transferred to another jurisdiction',
-    'Person Under Investigation (PUI)',
-    'Case confirmed',
-    'Past monitoring period',
-    'Meets criteria to discontinue isolation',
-    'Fully Vaccinated',
-    'Deceased',
-    'Duplicate',
-    'Other'
+    'Completed Monitoring', 'Meets criteria to shorten quarantine', 'Does not meet criteria for monitoring', 'Meets Case Definition',
+    'Lost to follow-up during monitoring period', 'Lost to follow-up (contact never established)', 'Transferred to another jurisdiction',
+    'Person Under Investigation (PUI)', 'Case confirmed', 'Past monitoring period',
+    *(['Meets criteria to discontinue isolation'] if ISOLATION_AVAILABLE),
```

Comment:
if we're going to put multiple values on each line, this one can fit on the line above

Reply:
One value per line is used in two arrays that follow (FOLLOW_UP_FLAG_REASONS and SYSTEM_SELECTABLE_MONITORING_REASONS). That seems like it would be easer to read, especially with the conditional inclusion here.

Reply:
@timwongj any thoughts?

Reply:
that's fine with me!